package-kuring/Sources/Caches/Spotlight.swift

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright (c) 2024 쿠링
+// See the 'License.txt' file for licensing information.
+//
+
+import Models
+import CoreSpotlight
+import ComposableArchitecture
```

Comment:
```suggestion
import Dependencies
```

테스트 실패 원인입니다.

Caches 는 TCA 까진 필요없고 `@Dependency` 만 필요하기 때문에 swift-dependencies의 Depedencies 모듈을 import 해서 사용합니다

Reply:
우선 수정 완료했습니다.

다만, 이 부분 CI쪽 살펴 보았는데, 잘 이해가 가지 않아서,, CI 더 조사해보겠습니다.

Reply:
두번째로 실행한 테스트 실패원인은 테스트 빌드는 성공했는데
Bookmark 메서드 내에서 Spotlight 관련 인터페이스가 추가하면서 실패하는 거 같네요.
한번 살펴볼게요

Reply:
XCTest 돌리면 자동으로 `@Dependency` 객체의 testValue 를 사용하게 되는데 Spotlight 에 testValue 가 선언되어 있지 않아서 그렇습니다.

`Bookmarks` 도 `testValue` 선언을 안했는데 테스트에러가 없는 이유는 제가 `@Dependency` 값을 쓴게 아니라 `Bookmarks.default` 객체를 직접 접근해서 사용했기 때문입니다. (실제 파일 저장을 테스트하기 위함)
```swift
try Bookmarks.default.add(Notice.random)
```